### PR TITLE
DELIA-62102 Update Thunder Security Logging to add to Telemetry

### PR DESCRIPTION
**Pull request by ramkumarpraba**
Reason for change : Easy to parse and be added to Telemetry.
Test Procedure: None
Priority: P1
Risks: None
Signed-off-by: Ramkumar_Prabaharan <Ramkumar_Prabaharan@comcast.com>

Change-Id: I19b3a40c13754cd886d0a7970b81bcf5412eb720

### CODE REVIEW

**Review by anand-ky — CHANGES_REQUESTED** [review]
Pls update changelog and patch version.

File: SecurityAgent/CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.1.1] - 2023-06-20
+### Changed
+- Update Thunder Security Logging to add to Telemetry
+
 ## [1.1.1] - 2023-06-05
```
Comment (ramkumarpraba) [inline SecurityAgent/CHANGELOG.md:23]: Hi @anand-ky Here already 1.1.1 there and API_VERSION_NUMBER_PATCH as 0 .
Now I neeed to update here 1.1.2 and API_VERSION_NUMBER_PATCH as 2 ?

Comment (anand-ky) [reply SecurityAgent/CHANGELOG.md:23]: yes pls update to 1.1.2

Comment (ramkumarpraba) [reply SecurityAgent/CHANGELOG.md:23]: Done anand

**Review by anand-ky — APPROVED** [review]

**Pull request merged**